.gitignore

```diff
@@ -141,3 +141,7 @@ tags
 # examples
 examples/*/data/
 examples/*/visualizations/
+/titanic/
```

Comment:
Not sure if the update to `.gitignore` is needed.

Reply:
@ShreyaR  Thank You for improvments :)  I 'm not experienced in contribution in projects but i will try do inverse yeo johnson function but I have to inverse equations:
```
y = ((x + 1)**lmbda - 1) / lmbda,                for x >= 0, lmbda != 0
    log(x + 1),                                  for x >= 0, lmbda = 0
    -((-x + 1)**(2 - lmbda) - 1) / (2 - lmbda),  for x < 0, lmbda != 2
    -log(-x + 1),                                for x < 0, lmbda = 2
```
I'm not sure how inverse function interact with output feature in ludwig. Sometimes good option is to predict output value  after transforation with yeo-johnson and after prediction convert with inverse function to original range.